src/components/orderList/OrderProduct.tsx

```diff
@@ -0,0 +1,75 @@
+import { OrderItem } from "api/orders";
+import React from "react";
+import { styled } from "styled-components";
+
+const OrderProduct = (order: OrderItem) => {
+  return (
+    <Wrapper>
+      <ImgBox src={order.product.imageUrl} alt={`${order.product.name} 상품 이미지`} />
```

Comment:
1. 이미지가 제대로 뜨지 않을때를 대비해서 기본 이미지를 보여주는 것이 있으면 좋겠어요. 
공통 이미지 컴포넌트를 만들어보는 건 어떨까요?
2. 아래 제목이 있으니 중복으로 `alt`를 써줄 필요는 없을 것 같아요.
```suggestion
      <ImgBox src={order.product.imageUrl} alt="" />
```

Reply:
여기선 불필요한 속성이였군요..!
이미지가 깨졌을 때도 고려해보겠습니다!